src/main/resources/public/js/src/core/validators.js

```diff
@@ -67,6 +67,14 @@ define(function (require) {
         }
         return null;
     };
+
+    var valueOutOfRange = function (val, options, Util) {
```

Comment:
There is no need of additional validation func, Use minMaxNumberRequired validator